x-pack/legacy/plugins/actions/server/builtin_action_types/server_log.ts

```diff
@@ -48,8 +52,9 @@ async function executor(
   const actionId = execOptions.actionId;
   const params = execOptions.params as ActionParamsType;
 
+  const sanitizedMessage = withoutControlCharacters(params.message);
   try {
-    logger[params.level](params.message);
+    logger[params.level](`server-log: ${sanitizedMessage}`);
```

Comment:
the prefix seems to be causing a test failure in:

https://github.com/elastic/kibana/blob/65de3c507c480a021416fea2b741c11c695ba291/x-pack/legacy/plugins/actions/server/builtin_action_types/server_log.test.ts#L97-L101

Reply:
ya, just pushed a commit with the fix